packages/flutter/lib/src/material/bottom_sheet_theme.dart

```diff
@@ -46,6 +47,12 @@ class BottomSheetThemeData extends Diagnosticable {
   /// If null, [BottomSheet] defaults to 0.0.
   final double elevation;
 
+  /// Value for [BottomSheet.backgroundColor] when the Bottom sheet is presented
+  /// as a modal bottom sheet.
+  ///
+  /// If null, [BottomSheet.backgroundColor] defaults to [backgroundColor].
```

Comment:
The defaulting rules for modal bottom sheets shouldn't be documented here. And line 464 in bottom_sheet.dart doesn't default to the theme's backgroundColor when modalBackgroundColor is null.

Reply:
Done